python/test/gluon/test_lowerings.py

```diff
@@ -136,11 +134,9 @@ def test_reduce_layouts(M, N, src_layout, axis, epilogue_kind, dtype_str, saniti
     if src_layout == "linear_layout":
         src_layout = ttgl.DistributedLinearLayout(reg_bases=[[0, 16], [1, 0], [2, 0], [4, 0], [8, 0], [16, 0]],  #
                                                   lane_bases=[[0, 0], [0, 1], [0, 2], [0, 4], [0, 8]],  #
-                                                  warp_bases=[[32, 0], [0, 32]], block_bases=[], shape=[M, N])
+                                                  warp_bases=[[32, 0], [0, 32]], block_bases=[], shape=[64, 64])
         if THREADS_PER_WARP != (1 << len(src_layout.lane_bases)):
             pytest.skip(f"Skipping. This LinearLayout assumes {1 << len(src_layout.lane_bases)} threads per warp")
-        elif M < 64 or N < 64:
-            pytest.skip(f"Skipping. This LinearLayout assumes M >= 64 and N >= 64, got M={M}, N={N}")
```

Comment:
Given that you are hardcoding the shape don't remove the skip as we don't want to execute the same test many times.